tiktok.cs

```diff
@@ -5,394 +5,105 @@ namespace First_project
     class Program
     {
             
-        static void XTurn(out int xGuessB, out int yGuessB)
+        static void Turn(out int xGuessB, out int yGuessB, int playerToken)
         {
             string xVal;
             string yVal;
-            Console.WriteLine("Player X! Are you ready?");
-            Console.WriteLine("What is the x coordinate of your guess?");
-            xVal = Console.ReadLine();
-            xGuessB = Convert.ToInt32(xVal);
-            Console.WriteLine("What is the y coordinate of your guess?");
-            yVal = Console.ReadLine();
-            yGuessB = Convert.ToInt32(yVal);
-
-        }
+            string player = playerToken == 1 ? "X" : "Y";
 
-
-        static void YTurn(out int xGuessB, out int yGuessB)
-        {
-            string xVal;
-            string yVal;
-            Console.WriteLine("Player Y! Are you ready?");
+            Console.WriteLine($"Player {player}! Are you ready?");
             Console.WriteLine("What is the x coordinate of your guess?");
             xVal = Console.ReadLine();
             xGuessB = Convert.ToInt32(xVal);
+            xGuessB--;
             Console.WriteLine("What is the y coordinate of your guess?");
             yVal = Console.ReadLine();
             yGuessB = Convert.ToInt32(yVal);
+            yGuessB--;
         }
 
         static void DrawBoard(int[,] board)
         {
-            Console.WriteLine($"{board[1, 3]} {board[2, 3]} {board[3, 3]}");
-            Console.WriteLine($"{board[1, 2]} {board[2, 2]} {board[3, 2]}");
-            Console.WriteLine($"{board[1, 1]} {board[2, 1]} {board[3, 1]}");
+            Console.WriteLine();
+            Console.WriteLine($"{board[0, 2]} {board[1, 2]} {board[2, 2]}");
+            Console.WriteLine($"{board[0, 1]} {board[1, 1]} {board[2, 1]}");
+            Console.WriteLine($"{board[0, 0]} {board[1, 0]} {board[2, 0]}");
+            Console.WriteLine();
         }
 
-        static bool CheckVictoryX(int[,] board, int xGuess, int yGuess)

```

Comment:
Again, you had CheckVictoryX and CheckVictoryY doing the same thing, so can just make one function.